### DISCUSSION TITLE
Error with wavelet_coherence when using freq_method='scale'

### DISCUSSION THREAD

**Discussion by LF-Earth** [LinkedEarth/Pyleoclim_util#462] [Q&A]
Hi, thank you for Pyleoclim. It is a really nice project. I have been learning Python, so I have learned a lot using it. 
When using this tutorial (http://linked.earth/PyleoTutorials/notebooks/L2_wavelet_analysis.html),
I tried to change the freq_method to scale: 
coh = co2ts.wavelet_coherence(dDts,method='wwz', freq_method='scale')
   
I received the following error: 

rect = np.zeros(length)
ValueError: negative dimensions are not allowed

Could you help me with that? If I try 'log', there is no problem.

**Comment by SHEN-Cheng** [comment]
This topic is quite similar to the issue: https://github.com/LinkedEarth/Pyleoclim_util/issues/623#issuecomment-2469397216

**Comment by CommonClimate** [comment]
Yes, this has been addressed by #635
